### DISCUSSION TITLE
Are we only using the GraphQL API to get data from the data base or does it also allow us to put/modify data in the database also?

### DISCUSSION THREAD

**Discussion by johnnyhchen** [gp-seattle/rms-app#94] [Backend Questions]
See title.

**Comment by yauj** [comment]
It allows us to put/modify data technically, but remember that to solve concurrency, the write APIs go through Lambda and DynamoDB directly, v.s. using the local database, or else multiple people can borrow on multiple devices at the same time for example.
TLDR; We only GraphQL for read operations.
